src/Request/Search/SearchImagesCreativeByImage.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace GettyImages\Api\Request\Search {
+
+use GettyImages\Api\Request\FluentRequest;
+use GettyImages\Api\Request\WebHelper;
+use Exception;
+
+class SearchImagesCreativeByImage extends FluentRequest {
+
+    /**
+     * @ignore
+     */
+    protected $route = "search/images/creative/by-image";
+
+    protected function getRoute() {
+        return $this->route;
+    }
+
+    protected function getMethod() {
+        return "get";
+    }
+
+    /**
+    * @ignore
+     */
+    private function addToBucket(string $filename, string $filepath)
```

Comment:
It would be good to name the parameters so it is clear which is the source and which is the destination.

Reply:
Updated